samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/assessment/PublishedItemData.java

```diff
@@ -603,6 +603,12 @@ public String getAnswerKey() {
 					if (!this.getTypeId().equals(TypeD.MATCHING)) {
 						if (this.getTypeId().equals(TypeD.TRUE_FALSE)) {
 							answerKey = a.getText();
+						} else if (this.getTypeId().equals(TypeD.CALCULATED_QUESTION)) {
+							if (("").equals(answerKey)) {
```

Comment:
Ok, I changed the code as you suggested. It has been necessary to include the dependency of commons-lang in the pom.xml of samigo-hibernate